hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/HapiTxnOp.java

```diff
@@ -783,9 +783,13 @@ public T txnId(String name) {
         return self();
     }
 
-    public T setNode(String account) {
-        node = Optional.of(HapiPropertySource.asAccount(account));
+    public T setNodeId(AccountID account) {
+        node = Optional.of(account);
+        return self();
+    }
 
+    public T setNode(String account) {
```

Comment:
Nit:
```suggestion
    public T setNode(String accountNum) {
```
Part of the difficulty with the test framework is that the term `account` can mean an account name, an account number, or a fully qualified account ID

Reply:
Fixed.